<!--  UBO Slam: /mapping_nodelet/pointcloud_downsampled    
      UBO new dataset 1:/mapping_nodelet/pointcloud
      UBO new dataset 2:/scan_assembler_nodelet/assembled_cloud
      NDT slam: /ndt_map-->
<launch>

  <node pkg="fused_terrain_classifier" type="fused_classifier" name="fused_classifier" output="screen">

<!--     <remap from="/points_raw" to="/surfel_map/pointcloud" /> -->
     <param name="image_transport" value="compressed" />

     <remap from="/points_raw" to="/mapping_nodelet/pointcloud_downsampled" />
     <remap from="/image_seg/compressed" to="/image/label_output/compressed" />
<!--  <remap from="/image_seg/compressed" to="/camera_right_hand/image_raw/compressed" />  -->

  </node>
  
</launch>
